mandatory/4-tax.js

```diff
@@ -17,7 +23,12 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  let tax = price * 0.2;
+  let result = tax + price;
+  result = result.toFixed(2);
+  return `£${result}`;
+}
```

Comment:
Great job  ashjan 😍
but I thought here you can call the previous function (calculateSalesTax)that you had written it before, instead of writing the code again

Reply:
thank you for comment 😍